src/ServiceControl.Audit.Persistence.RavenDb5/CustomChecks/CheckRavenDBIndexLag.cs

```diff
@@ -22,71 +20,70 @@ public CheckRavenDBIndexLag(IRavenDbDocumentStoreProvider documentStoreProvider)
         public override async Task<CheckResult> PerformCheck()
         {
             var store = documentStoreProvider.GetDocumentStore();
-            var statistics = await store.Maintenance.SendAsync(new GetIndexesStatisticsOperation());
-            var indexes = statistics.OrderBy(x => x.Name).ToArray();
+            var statistics = await store.Maintenance.SendAsync(new GetStatisticsOperation());
+            var indexes = statistics.Indexes.OrderBy(x => x.Name).ToArray();
 
-            CreateDiagnosticsLogEntry(indexes);
+            CreateDiagnosticsLogEntry(statistics, indexes);
 
             var indexCountWithTooMuchLag = CheckAndReportIndexesWithTooMuchIndexLag(indexes);
 
             if (indexCountWithTooMuchLag > 0)
             {
-                return CheckResult.Failed($"At least one index significantly stale. Please run maintenance mode if this custom check persists to ensure index(es) can recover. See the ServiceControl log file for more details. Visit https://docs.particular.net/search?q=servicecontrol+troubleshooting for more information.");
+                return CheckResult.Failed($"At least one index significantly stale. Please run maintenance mode if this custom check persists to ensure index(es) can recover. See log file for more details. Visit https://docs.particular.net/search?q=servicecontrol+troubleshooting for more information.");
             }
 
             return CheckResult.Pass;
         }
 
-        static int CheckAndReportIndexesWithTooMuchIndexLag(IndexStats[] indexes)
+        static int CheckAndReportIndexesWithTooMuchIndexLag(IndexInformation[] indexes)
         {
             int indexCountWithTooMuchLag = 0;
 
             foreach (var indexStats in indexes)
             {
-                // IndexingLag is the number of documents that the index is behind, it is not a time unit.
-                var indexLag = indexStats.Collections.Values.Sum(x => x.DocumentLag);
-                indexLag = Math.Abs(indexLag);
-
-                if (indexLag > IndexLagThresholdError)
-                {
-                    indexCountWithTooMuchLag++;
-                    _log.Error($"Index [{indexStats.Name}] IndexingLag {indexLag:n0} is above error threshold ({IndexLagThresholdError:n0}). Launch in maintenance mode to let indexes catch up.");
-                }
-                else if (indexLag > IndexLagThresholdWarning)
+                if (indexStats.IsStale && indexStats.LastIndexingTime.HasValue)
                 {
-                    indexCountWithTooMuchLag++;
-                    _log.Warn($"Index [{indexStats.Name}] IndexingLag {indexLag:n0} is above warning threshold ({IndexLagThresholdWarning:n0}). Launch in maintenance mode to let indexes catch up.");
+                    var indexLag = DateTime.UtcNow - indexStats.LastIndexingTime.Value; // TODO: Ensure audit ravendb5 persistence uses the same index lag behavior based on time
```

Comment:
This IS the audit instance :wink:
```suggestion
                    var indexLag = DateTime.UtcNow - indexStats.LastIndexingTime.Value;
```